### DISCUSSION TITLE
Issues arise when using key bindings such as <C-1> with Kitty and Zellij

### DISCUSSION THREAD

**Discussion by GiuseppeMP** [zellij-org/zellij#3511] [Q&A]
Hello everyone,

I am using this command to debug the send keys `kitten show-key -m kitty`, but as you can see in the image below, I am unable to use my binds <C-number> inside neovim while using Zellij. 

It's working when I use just kitty. (as you can see in the right)

I am not sure where to start investigating (nvim, zellij, kitty). Does anyone have any tips?

![image](https://github.com/user-attachments/assets/f88918f6-c6b8-4367-bf05-e2e0b1315773)

**Comment by swarnendubiswas** [comment]
I have a similar problem detecting "Alt+$" with Zellij + Alacritty/Kitty + Emacs. Vanilla Alacritty/Kitty + Emacs works. 

I encounter this problem with v0.42.1, irrespective of the value of `disable_kitty_keyboard_protocol.` Any tips on debugging the issue would be helpful.
